jenkins-integration/beaker/gatling/20_start_memory_watcher.rb

```diff
@@ -0,0 +1,27 @@
+test_name "Start memory watching script on SUT"
+
+simulation_id = ENV['GATLING_SIMULATION_ID']
+logger_refresh = ENV['MEMORY_WATCHER_REFRESH'] || 300
+
+if not simulation_id
+  abort "GATLING_SIMULATION_ID environment variable required"
+end
+
+remote_logger_dir = "/root/#{simulation_id}_memory_usage"
```

Comment:
Are you talking about the SUT or the gatling driver? I don't think we'll need `gatling-puppet-load-test` on the SUT (where this script is pointed)